<launch>
  <node name="ground_segmentation" pkg="ERPoT" type="ground_segmentation" output="screen">
    <rosparam command="load" file="$(find ERPoT)/config/obstacle_params.yaml" />
    <param name="cloud_topic" type="string" value="/velodyne_points" />
    <!-- hesai40p_points_xyzirt  /os1_cloud_node/points /velodyne_points -->
  </node>

  <node name="$(anon rviz)" pkg="rviz" type="rviz" args="-d $(find ERPoT)/rviz/demo.rviz"/>

</launch>
